doc/release/1.13.0-notes.rst

```diff
@@ -29,6 +29,9 @@ Future Changes
 Compatibility notes
 ===================
 
+``numpy.hstack()`` now throws ValueError instead of IndexError when input is empty.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I will change it. I didn't realize that is a heading line. I just thought that was a dividing line.